extension/src/test/suite/config.test.ts

```diff
@@ -159,6 +159,6 @@ suite('Config Test Suite', () => {
         config.getFocusedProjects(),
         'should exclude projects that are outside of the workspace'
       ).to.equal(undefined)
-    })
+    }).timeout(10000)
```

Comment:
This is fine as a hack, but there are 8 `expect` calls in that test; maybe breaking it up in multiple tests would help more than increasing the timeout.